software/revolt.yml

```diff
@@ -7,7 +7,7 @@ platforms:
   - Rust
 tags:
   - Communication - Custom Communication Systems
-source_code_url: https://github.com/revoltchat/revolt
+source_code_url: https://github.com/revoltchat/self-hosted
```

Comment:
```suggestion
source_code_url: https://github.com/revoltchat
```

Reply:
> Another alternative is https://github.com/revoltchat/backend (which actually is the backend source code, but doesn't contain installation instructions)

I thought about using the backend repo or another one but ended with the self-hosted guide since the codebase is spread across multiple repositories.